gateway_code/gateway_logging.py

```diff
@@ -28,16 +28,19 @@
 from logging.handlers import RotatingFileHandler
 
 # set default logger level to DEBUG to log everything
+import sys
```

Comment:
This sys module is not used in this module, it should be removed

Reply:
it seems to be used, sys.stdout is used for the stdout logging